packages/core/src/discovery.js

```diff
@@ -95,6 +98,45 @@ function parseDomResources({ url, domSnapshot }) {
   }, new Map([[rootResource.url, rootResource]]));
 }
 
+async function parseImageSrcset(snapshot, discovery) {
```

Comment:
```suggestion
async function gatherSrcsetResources(snapshot, discovery) {
```
Its not parsing srcset for image urls